app/services/taxonomy/build_taxon.rb

```diff
@@ -44,6 +47,10 @@ def content_item
       raise(TaxonNotFoundError, e.message)
     end
 
+    def validate_taxon_response!
+      raise(ContentTypeError) if content_item["document_type"] != "taxon"
```

Comment:
Given we also want to display the homepage, and that has a document type of `homepage`, this might need to be a little more permissive?

Reply:
Ah, yes! I totally overlooked the different homepage document type. Thanks for bringing it up

Reply:
Added in https://github.com/alphagov/content-tagger/pull/568/commits/b3a255810bd5e4b18ed85d1ade3b2a0b64059b30

Reply:
Looks good 👍